.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
what is the policy around deverloper's editor config files? i usually globally ignore my own.

Reply:
Ugh yeah I should do that as well, I'll remove this one. It always gets added by accident